### PR TITLE
9789 - added .module to quick-hits.scss file; changed to h3 heading i…

### PR DESCRIPTION
**Pull request by brianpetway**
…n header component; changed font size and weight on 'Share' text in Share component

https://federal-spending-transparency.atlassian.net/browse/DA-9789

### CODE REVIEW

File: src/templates/quick-hits.module.scss [hunk 1]
```diff
@@ -1,8 +1,8 @@
-.template-container {
+.templateContainer {
     margin-bottom: 100px;
 }
 
-.quick-hit-container {
+.quickHitsContainer {
```
Comment (anjenkin) [inline src/templates/quick-hits.module.scss:5]: Side note, scss modules recognizes class names with dashes as camelcase so you didn't have to change the classnames

File: src/components/quick-hits-header/quick-hits-header.module.scss [hunk 2]
```diff
@@ -10,7 +10,7 @@
 }
 
 .header {
-  @include h4-subheading();
+  @include h3-subheading();
```
Comment (anjenkin) [inline src/components/quick-hits-header/quick-hits-header.module.scss:13]: For 992px and under the header font size should be 32px (or 2rem).  With this fix, the header font is 40px for 992px and under

**Review by anjenkin — CHANGES_REQUESTED** [review]
See comments inline

File: src/templates/quick-hits.module.scss (hunk 1 above)
Comment (brianpetway) [reply src/templates/quick-hits.module.scss:5]: I know, it just bugs me when the same class is spelled differently in different places.

File: src/components/quick-hits-header/quick-hits-header.module.scss (hunk 2 above)
Comment (brianpetway) [reply src/components/quick-hits-header/quick-hits-header.module.scss]: Changed it. But this means it was correct before doesn't it?

**Comment by brianpetway** [comment]
Ready to review again.

**Review by anjenkin — APPROVED** [review]

**Pull request merged**